src/widget.cpp

```diff
@@ -1690,7 +1691,7 @@ std::string widget::layout( const avatar &ava, unsigned int max_width, int label
                         cur_width = cur_child._width;
                     }
                     // if last widget make it take the remaining space
-                    if( wid == wgts.back() ) {
+                    if( i == wgts.size() - 1 ) {
                         cur_width = avail_width - total_width;
```

Comment:
```suggestion
                        cur_width = std::max( 0, avail_width - total_width );
```
It would probably be good to clamp this at 0, since a negative value can cause an exception later on

Reply:
Done. Thanks!